package.json

```diff
@@ -44,6 +44,8 @@
   },
   "packageManager": "yarn@4.0.2",
   "dependencies": {
+    "@types/animejs": "^3.1.12",
```

Comment:
@types should be `devDependencies`

But how about adding this only on `apps/namadillo`?
So, it'll be easier to eject each project later :)

Reply:
Ops, thanks for the catch! This should lie on apps/namadillo! Will make this change